server/filters/format-date.js

```diff
@@ -32,17 +32,33 @@ export function formatDateRangeWithMessage({start, end}: {start: Date, end: Date
   const isPast = momentEnd.isBefore(momentNow);
   const isLastWeek = momentNow.isBetween(momentWeekBeforeEnd, momentEnd);
 
-  if (isFuture) {
+  return {
+    isFuture,
+    isPast,
```

Comment:
Could we not infer one from the other?
What about the `isPresent` state 🚎 ?

Reply:
Yeah, I was wondering about this – I might pin this to another PR.